### PR TITLE
#patch Fixing token permisions so that the Github Bot can push to main

### PR DESCRIPTION
**Pull request by jacobcrawford**

### CODE REVIEW

**Comment by jacobcrawford** [comment]
Commit Action needs a token for a user that can push to main.
https://github.com/stefanzweifel/git-auto-commit-action

File: .github/workflows/build_release.yml [hunk 1]
```diff
@@ -66,6 +67,7 @@ jobs:
         with: 
           commit_message: "New helm chart version ${{ steps.get-latest-tag.outputs.tag }}"
           tagging_message: ${{ steps.get-latest-tag.outputs.tag }}
+          push_options: --force
```
Comment (skovbakke) [inline .github/workflows/build_release.yml:70]: Why do we need to force it?

Comment (jacobcrawford) [reply .github/workflows/build_release.yml:70]: The docs state that we have to enable force push.

Comment (skovbakke) [reply .github/workflows/build_release.yml:70]: Do we use protected branches?

Comment (jacobcrawford) [reply .github/workflows/build_release.yml:70]: Yes since there are setup restrictions for collaborators on branches. E.g. we can not push to main.

Comment (skovbakke) [reply .github/workflows/build_release.yml:70]: Men har vi tilladt `force push` på main? og vil vi overhovedet det? (Det vil betyde at vi alle sammen kan udføre et force push på main uden noget review. Skulle vi ikke i stedet give github action bot tilladelse til at pushe til main? @MartinSchmidt Hvad siger du?

Comment (jacobcrawford) [reply .github/workflows/build_release.yml:70]: That is a good question.

**Pull request closed without merging**